oss/interval_tree/IntervalTree.h

```diff
@@ -67,7 +67,7 @@ namespace interval_tree
         };
 
         IntervalTree() :
-            left(nullptr), right(nullptr), center(0)
+            left(nullptr), right(nullptr), center(Scalar{})

```

Comment:
if center is already a `Scalar`:

```suggestion
            left(nullptr), right(nullptr), center()
```